qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -14,3 +14,6 @@
                 - memory_protection_key:
                     cpu_model_flags += ",+pku,check"
                     flags = "pku ospke"
+                - avx512_series_flags_5:
+                    cpu_model_flags += ",+avx512-bf16"
+                    flags = "avx512_bf16"
```

Comment:
Do you need restriction on guest? Does rhel7 guest work?

Reply:
Only rhel8.2.0 or later work and have updated, please help check.

 (1/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: CANCEL: The guest kernel doesn't support avx512_bf16 (2.20 s)
 (2/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: PASS (65.30 s)

Reply:
Why not `no RHEL.6, RHEL.7, RHEL.8.1`?

Reply:
> Why not `no RHEL.6, RHEL.7, RHEL.8.1`?

looks good, thanks and update.

Reply:
python3 ConfigTest.py --guestname=RHEL.8.0.0,RHEL.8.1.0,RHEL.8.2.0 --testcase=x86_cpu_flags.intel.avx512_series_flags_5 --clone=no

 (1/1) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: PASS (65.78 s)